kv/dist_sender.go

```diff
@@ -233,9 +233,11 @@ func (ds *DistSender) RangeLookup(key roachpb.RKey, desc *roachpb.RangeDescripto
 		Reverse:         useReverseScan,
 	})
 	replicas := newReplicaSlice(ds.gossip, desc)
+	trace := ds.Tracer.StartSpan("range lookup")
+	defer trace.Finish()
 	// TODO(tschottdorf) consider a Trace here, potentially that of the request
```

Comment:
could you reword the TODO and move it two lines up? Something along the lines of `// TODO(tschottdorf): Ideally we would use the trace of the request which caused this lookup instead of a new one.`